composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.2.6",
+        "dpc-sdp/tide_core": "dev-feature/SDPA-3505-turn-off-clamav-module-in-tide_core_module as 1.4.3",
```

Comment:
@vincent-gao , do you still need to do this, this PR is merged?

Reply:
@kurtfoster , this `dev-feature/SDPA-3505-turn-off-clamav-module-in-tide_core_module` will not be merged. I will release a new tide_core to be added.